functions/src/common/utils.ts

```diff
@@ -14,6 +14,31 @@
  * limitations under the License.
  */
 
+import {GroundProtos} from '@ground/proto';
```

Comment:
At some point we should rename this file to make it more clearly scoped. Ok for now.

Reply:
ok. noted.